.pre-commit-config.yaml

```diff
@@ -32,10 +32,6 @@ repos:
         args: [ --fix ]
       # Run the formatter.
       - id: ruff-format
-  - repo: https://github.com/python-poetry/poetry
-    rev: 1.6.0
-    hooks:
-      - id: poetry-check
```

Comment:
`rye` doesn't have a "check" command. `rye show --installed-deps` shows what is installed, and you could `diff requirements-dev.lock`, but you would have to ignore comment strings, `setuptools` and convert the lockfile to all lower case...